deploy/complete/docker-compose/README.md

```diff
@@ -1,21 +1,50 @@
 # MuShop Docker
 
-The complete **MuShop** application can be run using `docker-compose`
+The complete **MuShop** application can be run using `docker-compose` locally.
+Like most 12-factor micro services, **MuShop** is built with testability 
+and portability in mind. The application configuration is [externalized](https://12factor.net/config) 
+from the container image so that the same application container image can be configured 
+to behave differently.  
 
-## Prerequisites
+Using `docker-compose` to run **MuShop** is useful when making changes to individual services or 
+running tests on the application locally. If you have a kubernetes cluster available, the helm 
+based deployment is recommended. Note that you can have a local kubernetes cluster based on [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) or [kind](https://kind.sigs.k8s.io/)
```

Comment:
I'm inclined to suggest we omit this introduction altogether.

Reply:
I thought that we needed some context to reinforce that the container images are the same whether the user chooses the helm based installation or this. However the main reason someone would want to use this is to test quickly.

I am okay to remove this if you still think its unnecessary. The whole intro and the Prereq stuff or just the new content under prereq?